langchain/law_vectorstore.py

```
@@ -0,0 +1,496 @@
+        """처리 상태 JSON 파일 경로 반환"""
+        return os.path.join(self.persist_directory, "processed_states.json")
+
+    def _load_processed_states(self) -> dict:
+        """처리 상태 JSON 파일 로드"""
+        states_path = self._get_processed_states_path()
+
+        if os.path.exists(states_path):
+            try:
+                with open(states_path, "r", encoding="utf-8") as f:
+                    return json.load(f)
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)